unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx

```diff
@@ -108,19 +108,29 @@ export const EmailTemplatePreview = ({
       eventTime: '{Event time}',
       eventAddress: '{Event address }',
       // certificate details
-      certificationDetail: 'https://example.it',
+      certificationDetail: 'https://example.it', // TOFIX @kali: What is this?
```

Comment:
Same here!

Reply:
this is only a placeholder, changing to keep same format as above